hrqb/cli.py

```diff
@@ -39,3 +42,83 @@ def ping(ctx: click.Context) -> None:
             timedelta(seconds=perf_counter() - ctx.obj["START_TIME"]),
         ),
     )
+
+
+@click.group()
+@click.option(
+    "-p",
+    "--pipeline",
+    type=str,
+    required=True,
+    help="Pipeline Task class name to be imported from configured pipeline module, "
+    "e.g. 'MyPipeline'",
+)
+@click.option(
+    "-pm",
+    "--pipeline-module",
+    type=str,
+    required=False,
+    help="Module where Pipeline Task class is defined. Default: 'hrqb.tasks.pipelines'.",
+    default="hrqb.tasks.pipelines",
+)
+@click.option(
+    "--pipeline-parameters",
+    callback=click_argument_to_dict,
+    help="Comma separated list luigi Parameters to pass to HRQBPipelineTask, "
+    "e.g. 'Param1=foo,Param2=bar'.",
+)
+@click.pass_context
+def pipeline(
+    ctx: click.Context,
+    pipeline: str,
+    pipeline_module: str,
+    pipeline_parameters: dict,
+) -> None:
+    pipeline_task = HRQBPipelineTask.init_task_from_class_path(
+        pipeline,
+        task_class_module=pipeline_module,
+        pipeline_parameters=pipeline_parameters,
+    )
+    message = f"Successfully loaded pipeline: '{pipeline_module}.{pipeline}'"
+    ctx.obj["PIPELINE_TASK"] = pipeline_task
+    logger.debug(message)
+
+
+main.add_command(pipeline)
+
+
+@pipeline.command()
+@click.pass_context
+def status(ctx: click.Context) -> None:
+    pipeline_task = ctx.obj["PIPELINE_TASK"]
+    logger.info(pipeline_task.pipeline_as_ascii())
+
+
+@pipeline.command()
+@click.pass_context
+def remove_data(ctx: click.Context) -> None:
+    pipeline_task = ctx.obj["PIPELINE_TASK"]
+    logger.warning("Removing all Pipeline Tasks Targets (data).")
+    logger.info(pipeline_task.remove_pipeline_targets())
+    logger.info("Successfully removed pipeline artifacts.")
+
+
+@pipeline.command()
+@click.option(
+    "--remove-data",
+    "remove_artifacts",  # renamed as variable to avoid collision with other command
```

Comment:
I like this.  I was aiming for symmetry between the flag name and the actual CLI command it would run, but I think "cleanup" accurately reflects that it's something you do _after_ a run.